analysis_schema/pydantic_schema_example.json

```diff
@@ -5,11 +5,11 @@
       "ProjectionPlot": {
         "Dataset": [
           {
-            "FileName": "../../Data/IsolatedGalaxy/galaxy0030/galaxy0030",
+            "FileName": "IsolatedGalaxy/galaxy0030/galaxy0030",
             "DatasetName": "IG"
           },
           {
-            "FileName": "../../Data/enzo_tiny_cosmology/DD0000/DD0000",
+            "FileName": "enzo_tiny_cosmology/DD0000/DD0000",
```

Comment:
@samwalkow I changed these lines so that it's less system-dependent. But you can set up your yt config to look in your `Data` directory for files. You should have a `~/.config/yt/yt.toml` file, to which you can add: 

```
[yt]
test_data_dir = "/any/directory/that/you/want"
```

And then yt will always look in that directory for data.

Reply:
This is good to know and SUPER helpful!